README.md

```diff
@@ -402,6 +414,9 @@ Complete argument list:
         Prints the current version number of this tool.
     --verbose
         Writes detailed progress information to stderr.
+    --proxy=xxx
```

Comment:
Perhaps would be better to use ``<ip>:<port>`` notation in the example, besides that, lgtm.